mods/default/nodes.lua

```diff
@@ -453,14 +453,21 @@ minetest.register_node("default:dirt_with_grass_footsteps", {
 	}),
 })
 
-minetest.register_node("default:dirt_with_dry_grass", {
-	description = "Dirt with Dry Grass",
+minetest.register_node("default:dry_dirt", {
+	description = "Savanna Dirt",
+	tiles = {"default_dry_dirt.png"},
+	groups = {crumbly = 3, soil = 1},
+	sounds = default.node_sound_dirt_defaults(),
+})
+
+minetest.register_node("default:dry_dirt_with_dry_grass", {
+	description = "Savanna Dirt with Dry Grass",
 	tiles = {"default_dry_grass.png",
-		"default_dirt.png",
-		{name = "default_dirt.png^default_dry_grass_side.png",
+		"default_dry_dirt.png",
+		{name = "default_dry_dirt.png^default_dry_grass_side.png",
 			tileable_vertical = false}},
 	groups = {crumbly = 3, soil = 1, spreading_dirt_type = 1},
-	drop = 'default:dirt',
+	drop = 'default:dry_dirt',
```

Comment:
Sorry to be perfectionist but we are changing all ' to " in node names

Reply:
done